pkg/shared/utilitySQL/query_construction.go

```diff
@@ -1,16 +1,19 @@
-package utilitySQL
-
-import (
-	"github.com/google/uuid"
-
-	"github.com/cmsgov/mint-app/pkg/shared/utilityUUID"
-)
-
-// CreateIDQueryMap creates a map with key "id" which maps to the passed in id
-func CreateIDQueryMap(id uuid.UUID) map[string]interface{} {
-	id = utilityUUID.ValueOrNewUUID(id)
-
-	return map[string]interface{}{
-		"id": id,
-	}
-}
+package utilitySQL
+
+import (
+	"github.com/google/uuid"
+)
+
+// CreateIDQueryMap creates a map with key "id" which maps to the passed in id
+func CreateIDQueryMap(id uuid.UUID) map[string]interface{} {
+	return map[string]interface{}{
+		"id": id,
+	}
+}
+
+// CreateModelPlanIDQueryMap creates a map with key "model_plan_id" which maps to the passed in id
+func CreateModelPlanIDQueryMap(id uuid.UUID) map[string]interface{} {

```

Comment:
This is fine, I think leaving the content of the function inline might be clearer code, since the function isn't doing very much.